action_server/src/sema4ai/action_server/_protocols.py

```diff
@@ -183,3 +191,59 @@ class ArgumentsNamespaceStart(ArgumentsNamespaceBaseImportOrStart):
     expose: bool
     expose_allow_reuse: bool
     api_key: str
+
+
+class ArgumentsNamespacePackageOrganizations(ArgumentsNamespace):
+    command: Literal["package"]
+    package_command: Literal["organizations"]
+    access_credentials: Optional[str]
+    json: bool
+
+
+class ArgumentsNamespacePackagePush(ArgumentsNamespace):
+    command: Literal["package"]
+    package_command: Literal["publish"]
+    access_credentials: Optional[str]
+    organization_id: str
+    package_path: str
+    json: bool
+
+
+class ArgumentsNamespacePackageStatus(ArgumentsNamespace):
+    command: Literal["package"]
+    package_command: Literal["state"]
+    access_credentials: Optional[str]
+    organization_id: str
+    package_id: str
+    json: bool
+
+
+class ArgumentsNamespacePackageChangelog(ArgumentsNamespace):
+    command: Literal["package"]
+    package_command: Literal["changelog"]
```

Comment:
Maybe it should be `action-server package set-changelog`? Just by reading it I'm not sure if it's just querying or setting...

As a note, thanks for keeping the styling here (making proper typing is a bit boring, but it does help in knowing what to expect from the command line just from the types). Nice job.